lib/embed/embed.dart

```diff
@@ -96,6 +96,25 @@ class PlaygroundMobile {
     });
   }
 
+  /**
+   * Return if strong mode should be enabled
+   */
+  bool _isStrongMode(String strong) {
```

Comment:
I think the point of this function is to parse a query parameter to work out if it implies that strong mode should be enabled? If so, can we update the method and field name to make it more explicit?

Something like _parseStrongModeParam(String strongModeValueString)